front/app/containers/Admin/projects/components/ProjectRow/index.tsx

```diff
@@ -118,14 +128,13 @@ const ProjectRow = ({
                   buttonStyle="secondary"
                   icon={action.icon}
                   processing={action.processing}
-                  disabled={isBeingDeleted}
                 >
                   {action.buttonContent}
                 </RowButton>
               );
             }
           })}
-          {showMoreActions && (
+          {!hideMoreActions && userCanModerateProject && (
```

Comment:
I wonder if we should rather repeat the permissions check inside ProjectMoreActionsMenu to be really clear about it 🤔

Reply:
Indeed. I reviewed top to bottom and just seeing this at the end. I'm not sure about duplicating but I think that component should handle it's display logic (permissions). How it is used can be something separate. Say we can hide it for some instances if there is need but it inherently should handle those permissions

Reply:
Yes, I'm moving this back inside the component.